osu.Game/Rulesets/Mods/Mod.cs

```diff
@@ -136,19 +133,35 @@ public virtual Mod CreateCopy()
             // Copy bindable values across
             foreach (var (_, prop) in this.GetSettingsSourceProperties())
             {
-                var origBindable = prop.GetValue(this);
-                var copyBindable = prop.GetValue(copy);
-
-                // The bindables themselves are readonly, so the value must be transferred through the Bindable<T>.Value property.
-                var valueProperty = origBindable.GetType().GetProperty(nameof(Bindable<object>.Value), BindingFlags.Public | BindingFlags.Instance);
-                Debug.Assert(valueProperty != null);
+                var origBindable = (IBindable)prop.GetValue(this);
+                var copyBindable = (IBindable)prop.GetValue(copy);
 
-                valueProperty.SetValue(copyBindable, valueProperty.GetValue(origBindable));
+                // we only care about changes that have been made away from defaults.
+                if (!origBindable.IsDefault)
+                    copy.CopyAdjustedSetting(copyBindable, origBindable);
             }
 
             return copy;
         }
 
+        /// <summary>
+        /// When creating copies or clones of a Mod, this method will be called to copy explicitly adjusted user settings.
+        /// The base implementation will transfer the value via <see cref="Bindable{T}.Parse"/> and should be called unless replaced with custom logic.
+        /// </summary>
+        /// <param name="bindable">The target bindable to apply the adjustment.</param>
+        /// <param name="value">The adjustment to apply.</param>
+        internal virtual void CopyAdjustedSetting(IBindable bindable, object value)
```

Comment:
mind if i rename to `target` and `source`? kinda confusing to read otherwise. with the rename at least the rider param name hints help some.

Reply:
sounds better, yes.